core-kotlin-modules/core-kotlin-strings-4/src/test/kotlin/com/baeldung/sentencereverse/ReverseSentenceUnitTest.kt

```diff
@@ -0,0 +1,78 @@
+package com.baeldung.sentencereverse
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.util.*
+
+class ReverseSentenceUnitTest {
+
+    @Test
+    fun `reverse sentence using programmatic approach`() {
+        val sentence = "this is a complete sentence"
+        assertEquals("sentence complete a is this", reverseWordsInSentenceCustomMethod(sentence))
+    }
+
+    @Test
+    fun `reverse sentence using split and joinToString methods`() {
+        val sentence = "this is a complete sentence"
+        assertEquals("sentence complete a is this", reverseSentenceUsingReverseAndJoinToStringMethods(sentence))
+    }
+
+    @Test
+    fun `reverse sentence using a fold method`() {
+        val sentence = "this is a complete sentence"
+        assertEquals("sentence complete a is this", reverseSentenceUsingFoldMethod(sentence))
+    }
+
+    @Test
+    fun `reverse sentence using a stack`() {
+        val sentence = "this is a complete sentence"
+        assertEquals("sentence complete a is this", reverseWordsInSentenceUsingStack(sentence))
+    }
+
+    @Test
+    fun testReverseWordsInSentence() {
+        assertEquals("this is a complete sentence", reverseWordsInSentence("sentence complete a is this"))
+    }
```

Comment:
It looks like this example didn't actually make it to the article, so let's take it out.